apollo-router/src/configuration/migrations/0012-persisted-queries-ga.yaml

```diff
@@ -0,0 +1,5 @@
+description: Persisted queries are no longer preview, `preview_persisted_queries` is renamed `persisted_queries`
```

Comment:
Text chosen for consistency with 0008, though I would probably use a semicolon and add "to" after "renamed"